itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -3497,6 +3498,57 @@ public void testAuthForNotificationAPIs() throws Exception {
     }
   }
 
+  @Test
+  public void testRecycleFileDropTempTable() throws IOException {
+    String dbName = createDB(testName.getMethodName(), driver);
+
+    run("CREATE TABLE " + dbName + ".normal(a int)", driver);
```

Comment:
Why do we need this extra steps to create, insert and drop normal table?

Reply:
this is just to check the file is getting created in valid scenario

Reply:
valid scenario would be checked by other tests. Think, it is not needed here.